io.github.kovzol.bibref.yaml

```diff
@@ -0,0 +1,44 @@
+id: io.github.kovzol.bibref
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: bibref-qt
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+modules:
+  - name: sword
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: https://www.crosswire.org/ftpmirror/pub/sword/source/v1.9/sword-1.9.0.tar.gz
+        sha256: 42409cf3de2faf1108523e2c5ac0745d21f9ed2a5c78ed878ee9dcc303426b8a
+  - name: boost
+    buildsystem: simple
+    build-commands: ['./bootstrap.sh --prefix=/app --with-libraries=system,filesystem', './b2 -j $FLATPAK_BUILDER_N_JOBS install']
```

Comment:
please use proper yaml syntax. it's unreadable otherwise.

Reply:
I will fix this in a new commit.